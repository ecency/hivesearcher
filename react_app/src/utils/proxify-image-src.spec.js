/* eslint-disable */

import proxifyImageSrc from './proxify-image-src';

describe('Proxify image src', () => {
  it('(1) should proxify image src', () => {
    const input = `https://i.imgur.com/muESb0B.png`;

    expect(proxifyImageSrc(input)).toMatchSnapshot();
  });

  it('(2) should not proxify if already proxified', () => {
    const input = `https://steemitimages.com/0x0/https://steemitimages.com/DQmWK9ACVoywHPBJQdoTuJpoTSoaubBSKSAdZaJtw1cfLb9/adsactlywitness.gif`;

    expect(proxifyImageSrc(input)).toMatchSnapshot();
  });
});
