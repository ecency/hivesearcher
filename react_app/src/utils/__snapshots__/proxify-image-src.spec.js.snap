// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Proxify image src (1) should proxify image src 1`] = `"https://steemitimages.com/0x0/https://i.imgur.com/muESb0B.png"`;

exports[`Proxify image src (2) should not proxify if already proxified 1`] = `"https://steemitimages.com/0x0/https://steemitimages.com/DQmWK9ACVoywHPBJQdoTuJpoTSoaubBSKSAdZaJtw1cfLb9/adsactlywitness.gif"`;
